frontend/app/(tabs)/history.tsx

```diff
@@ -0,0 +1,267 @@
+    default:
+      return { icon: Trash2, color: 'text-gray-500', bgColor: 'bg-gray-500' };
+  }
+};
+
+const ScannedItem = ({
+  item,
+  onPress,
+}: {
+  item: ScannedItemType;
+  onPress: () => void;
+}) => {
+  const { icon: Icon, color, bgColor } = getIconAndColor(item.type);
+
+  return (
+    <TouchableOpacity
+      onPress={onPress}
+      className='mb-2 flex-row items-center rounded-lg bg-card p-4 shadow-sm'
+    >
+      <View
+        className={cn(
+          'mr-4 h-12 w-12 items-center justify-center rounded-full',
+          bgColor,
+        )}
+      >
+        <Icon size={24} color='white' />
+      </View>
+      <View className='flex-1'>
+        <Text className='text-lg font-semibold'>{item.title}</Text>
+        <Text className='text-sm text-gray-600'>{item.description}</Text>
+        <Text className='mt-1 text-xs text-gray-400'>{item.date}</Text>
+      </View>
+      <View className={cn('h-3 w-3 rounded-full', bgColor)} />
+    </TouchableOpacity>
+  );
+};
+
+const CallToActionButton = ({ onPress }: { onPress: () => void }) => (
+  <TouchableOpacity
+    onPress={onPress}
+    className='mt-4 flex-row items-center justify-center rounded-lg bg-blue-500 p-4'
+    accessibilityRole='button'
+    accessibilityLabel='Scan new item'
+  >
+    <CameraIcon className='mr-2 size-6 text-primary' />
+    <Text className='text-lg font-semibold text-primary'>Scan New Item</Text>
+  </TouchableOpacity>
+);
+
+export default function HistoryTab() {
+  const [scannedItems, setScannedItems] = useState<ScannedItemType[]>([]);
+  const [selectedItem, setSelectedItem] = useState<ScannedItemType | null>(null);
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const animatedIndex = useSharedValue<number>(0);
+  const animatedPosition = useSharedValue<number>(0);
+  // ref
+  const bottomSheetModalRef = useRef<BottomSheetModal>(null);
+
+  // variables
+  const snapPoints = useMemo(() => ['50%', '75%', '90%'], []);
+
+  // callbacks
+  const handlePresentModalPress = useCallback(() => {
+    bottomSheetModalRef.current?.present();
+    setIsOpen(true);
+  }, []);
+
+  const handleSheetChanges = useCallback((index: number) => {
+    console.log('handleSheetChanges', index);
+  }, []);
+
+  const handleScanNewItem = () => {
+    router.push('/scan');
+  };
+
+  const loadScannedItems = async () => {
+    try {
+      const storedItems = await AsyncStorage.getItem('scannedItems');
+      if (storedItems) {
+        setScannedItems(JSON.parse(storedItems));
+      }
+    } catch (error) {
+      console.error('Error loading scanned items:', error);
+    }
+  };
+
+  React.useEffect(() => {
+    // Load scanned items initially
+    loadScannedItems();
+
+    // Listen for new scanned items using mitt's "on" method
+    const updateScannedItemsListener = () => {
+      loadScannedItems();
+    };
+
+    scannedItemsEmitter.on('newItemScanned', updateScannedItemsListener);
```

Comment:
I think using an emitter is a bit overkill, because the user will never be on both the history page and the scanner page at the same time.

A better approach would be to fetch the saved items when the user enters the history page from the local storage (ideally, we would cache it so that it would only fetch when changes are made).